DependencyInjection/Configuration.php

```diff
@@ -963,4 +1062,119 @@ public function getConfigTreeBuilder()
 
         return $treeBuilder;
     }
+
+    private function handlerTypeAcceptedOptionsValidation(array $v)
+    {
+        if (!array_key_exists('type', $v)) {
+            return $v;
+        }
+
+        if (!array_key_exists(strtolower($v['type']), $this->acceptedParamsByHandlerType)) {
```

Comment:
Similarly to the note below, strtolower here and at line 1080 probably should be removed.. types were not case insensitive until now?